x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/zeek/zeek_details.test.tsx

```diff
@@ -113,7 +113,6 @@ describe('ZeekDetails', () => {
           />
         </TestProviders>
       );
-      expect(wrapper).toMatchSnapshot();
```

Comment:
@angorayc the other snapshot test above does `expect(wrapper.find('ZeekDetails')).toMatchSnapshot()`. If this test has value, doing the same would prevent us from having to remove the test completely.

Reply:
Thanks @rylnd, that's a good idea. Actually I added that line for manual debugging in my previous PR and forgot to remove that.
https://github.com/elastic/kibana/commit/e97b451cc186e47a4621b85cf2bd814a8529c0b6#r37522518

@FrankHassanabad would you want me to move this snapshot test to another block and apply ` expect(wrapper.find('ZeekDetails')).toMatchSnapshot()` or it is ok if I just remove this check?

Reply:
I think the team consensus has been that snapshots are optional but no one really cares if we delete them if there are other tests that provide coverage.

So, yeah, you are fine to delete this snapshot test if you want to. They don't provide us with a lot of value compared to focused testing.

But if you find value in them and want to continue using them (as long as they are not _huge_ outputs) that is fine as well.

So, in short, optionally you can delete any ones you want to or keep using them as long as they are not large snapshots.

Reply:
ok, given that I added it accidentally, removing it here. Thank you @rylnd @FrankHassanabad